mmv1/third_party/terraform/services/logging/resource_logging_project_sink_test.go

```diff
@@ -294,8 +294,6 @@ resource "google_logging_project_sink" "basic" {
   project     = "%s"
   destination = "storage.googleapis.com/${google_storage_bucket.log-bucket.name}"
   filter      = "logName=\"projects/%s/logs/compute.googleapis.com%%2Factivity_log\" AND severity>=ERROR"
-
-  unique_writer_identity = false
```

Comment:
Do you still have configs that set `unique_writer_identity = false`? Just want to make sure we cover the case of setting negative value even we changed the default value in TF

Reply:
yes, we have, for example: resource "google_logging_project_sink" "described"